src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,65 @@
 package mate.jdbc;
 
+import java.util.ArrayList;
+import java.util.List;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.CarService;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        Driver arthurDriver = new Driver("Arthur", "666");
+        Driver madhooDriver = new Driver("Madhoo", "1933");
+        {
+            DriverService driverService = (DriverService) injector.getInstance(DriverService.class);
+            driverServiceTest(driverService, arthurDriver);
+            driverServiceTest(driverService, madhooDriver);
+            driverService.getAll().forEach(System.out::println);
+        }
```

Comment:
```suggestion
            DriverService driverService = (DriverService) injector.getInstance(DriverService.class);
            driverServiceTest(driverService, arthurDriver);
            driverServiceTest(driverService, madhooDriver);
            driverService.getAll().forEach(System.out::println);
```

Let's remove those blocks.

It's better to separate logically grouped lines of code by **empty blocks (empty lines)** for the sake of readability and easier understanding either in the future or by other devs.

Reply:
Blocks are also clean code and readable, with a clear scope, fewer windows of vulnerability, and less complexity of imaginary actions. Т_т